docs/docsite/rst/scenario_guides/integration_guides.rst

```diff
@@ -0,0 +1,39 @@
+.. _integration_guides:
+
+******************
+Integration Guides
+******************
+
+The guides in this section cover integrating Ansible with a variety of
+platforms, products, and technologies. They explore particular use cases in greater depth and provide a more "top-down" explanation of some basic features.
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Cloud Integration Guides
+
+   guide_alicloud
+   guide_aws
+   guide_azure
+   guide_gce
+   guide_online
+   guide_packet
+   guide_rax
+   guide_scaleway
+   guide_vultr
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Network Integration Guides
+
```

Comment:
Does the main network index page want to be linked here?

Reply:
If you mean moving the main network index page to be under this Integrations sections, no. Network automatin has to stay at the highest level to remain visible.  If you mean should we have a separate link somewhere on the 'network integrations guide' section to point to the network main guides..possibly yeah.

Reply:
Sorry, I want clear. Would it be worth adding a link to the network guides (which shouldn't move)

Reply:
I am in favor of the idea - generally the easier it is for a user to get to the content she needs/wants, the better, so linking between related but different content is a good idea. I'm not sure of the best way to add a link to a TOC page, but I will experiment with options.